var/spack/repos/builtin/packages/py-tap-py/package.py

```diff
@@ -9,14 +9,18 @@
 class PyTapPy(PythonPackage):
     """Python TAP interface module for unit tests"""
 
-    homepage = "https://github.com/mblayman/tappy"
-    url      = "https://pypi.io/packages/source/t/tap.py/tap.py-1.6.tar.gz"
+    homepage = "https://github.com/python-tap/tappy"
+    url      = "https://pypi.io/packages/source/t/tap.py/tap.py-3.0.tar.gz"
 
-    version('1.6', sha256='3ee315567cd1cf444501c405b7f7146ffdb2e630bac58d0840d378a3b9a0dbe4')
+    version('3.0',   sha256='f5eeeeebfd64e53d32661752bb4c288589a3babbb96db3f391a4ec29f1359c70')
+    version('2.6.2', sha256='5f219d92dbad5e378f8f7549cdfe655b0d5fd2a778f9c83bee51b61c6ca40efb')
+    version('1.6',   sha256='3ee315567cd1cf444501c405b7f7146ffdb2e630bac58d0840d378a3b9a0dbe4')
 
     extends('python', ignore='bin/nosetests|bin/pygmentize')
 
-    depends_on('python@2.6:2.8,3.2:3.4')
-    depends_on('py-nose', type=('build', 'run'))
-    depends_on('py-pygments', type=('build', 'run'))
-    depends_on('py-setuptools', type='build')
+    depends_on('python@3.5:3.7', when='@3.0:')
+    depends_on('python@2.7:2.8,3.5:3.7', when='@2.6')
+    depends_on('python@2.6:2.8,3.2:3.4', when='@:1.8')
```

Comment:
I would remove the upper version constraint on these Python versions. They likely work with Python 3.8 even if the authors haven't yet tested them.

Reply:
I kept the upper limits since the changelog for the unreleased version 3.1 mentions "add support for Python 3.8", so I just assumed that I does not work with earlier versions.

Reply:
In that case maybe we can add 3.1 too?

Reply:
3.1 has not been released yet. I could add the master branch if you think it's necessary.

Reply:
Nah, don't worry about it. I think we're still a ways off from making Python 3.8 the default in Spack. I plan on switching whenever Anaconda does.